interface/src/__snapshots__/draganddrop.test.js.snap

```diff
@@ -0,0 +1,65 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`Test Dropzone component should render correctly with no props 1`] = `
+<div
+  className="container"
+>
+  <div
+    className="dropzone"
+    onBlur={null}
+    onClick={[Function]}
+    onDragEnter={[Function]}
+    onDragLeave={[Function]}
+    onDragOver={[Function]}
+    onDrop={[Function]}
+    onFocus={null}
+    onKeyDown={null}
+    style={
```

Comment:
These styles should be in another css file specific for testing; unless this file is autogenerated.